test/tests/functional/pbs_preemption.py

```diff
@@ -388,10 +394,15 @@ def test_preempt_retry(self):
 """
         self.insert_checkpoint_script(abort_script)
```

Comment:
just curious, why are we inserting the checkpoint abort script here? Don't we want the checkpoint to fail again, and for preemption to happen via method 'R' ?

Reply:
It doesn't really matter.  The point of this test is to retry preemption.  Try once, fail, and then try a second time.  I modified this test to trigger the invalid read.  If I wrote a different test for that, it'd be very close to this one.  The invalid read happens when there are multiple preemption methods.  This test used to only have C.  I modified this to use CR.  The second attempt needs to succeed.  It doesn't really matter which preemption method is successful.  I guess if you want I can leave the checkpoint abort script alone and allow the requeue to succeed.

Reply:
Ok, thanks for explaining. From the description of the test and how it currently behaves, it's kind of hard to understand why it bothers about the rerun aspect when, functionally, it's only exercising the checkpoint method. You also set 'rerun' to 'y' just before triggering the sched cycle, that seems to convey that we are expecting the R method to be exercised. So, I think that you should either add a comment to explain why you need the R to be configured, or just let checkpoint fail again and R be exercised.

Reply:
I didn't look at the block comment.  I should probably rewrite it some.  I'll also do as you say and leave checkpoint as failing and let requeue succeed.